src/Grids/vertically_stretched_rectilinear_grid.jl

```diff
@@ -174,11 +174,14 @@ short_show(grid::VerticallyStretchedRectilinearGrid{FT, TX, TY, TZ}) where {FT,
     "VerticallyStretchedRectilinearGrid{$FT, $TX, $TY, $TZ}(Nx=$(grid.Nx), Ny=$(grid.Ny), Nz=$(grid.Nz))"
 
 function show(io::IO, g::VerticallyStretchedRectilinearGrid{FT, TX, TY, TZ}) where {FT, TX, TY, TZ}
+    Δz_min = minimum(view(g.Δzᵃᵃᶜ, 1:g.Nz))
+    Δz_max = maximum(view(g.Δzᵃᵃᶜ, 1:g.Nz))
```

Comment:
I wonder if it makes sense to take the min/max over the spacings between the faces since they could potentially be larger than the spacings between the centers?

```suggestion
    Δz_min = minimum(view(g.Δzᵃᵃᶠ, 1:g.Nz+1))
    Δz_max = maximum(view(g.Δzᵃᵃᶠ, 1:g.Nz+1))
```

Alternatively, we could go over both the centers and faces but this could be wasteful?

```julia
    Δz_min = min(minimum(view(g.Δzᵃᵃᶜ, 1:g.Nz)), minimum(view(g.Δzᵃᵃᶠ, 1:g.Nz+1)))
    Δz_max = max(maximum(view(g.Δzᵃᵃᶜ, 1:g.Nz)), maximum(view(g.Δzᵃᵃᶠ, 1:g.Nz+1)))
```

Reply:
I thought about that, but my thinking is that it's best to err on the side of simplicity since the max/min between faces and centers are generally (always?) gonna be pretty similar. And this is way of printing is meant (IMO) to be kind of a bird's eye view of the whole thing anyway.

Also, in the tests I've run here, the min around centers was always smaller than the min around faces and the max around centers was also always larger than the max around faces!

That said, I'm happy to change it if you want.

Reply:
Ah no that makes sense then! Please ignore the suggestion.